server/Customer.java

```diff
@@ -4,15 +4,19 @@
 
 public class Customer {
 
-	private ArrayList<Account> accounts;
+	private ArrayList<newbank.server.Account> accounts;
```

Comment:
The `newbank.server` is not necessary, and actually makes the code harder to read, please remove it